Vagrantfile

```diff
@@ -35,16 +35,18 @@ def setup_host
 end
 
 # Make sure the host computer is set up every time a vagrant command is run
-setup_host
-
+if devconfig["use_vpn"]
+    setup_host
+end
 
-VAGRANTFILE_API_VERSION = "2"
 
-Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+Vagrant.configure("2") do |config|
```

Comment:
Why change this to hard coded?

Reply:
It gets rid of a warning `warning: already initialized constant VAGRANTFILE_API_VERSION`.  It also reflects the documented example at https://www.vagrantup.com/docs/vagrantfile/version.html.